pkg/diagnose/runner.go

```diff
@@ -522,8 +521,5 @@ func RegisterConnectivityDatadogEventPlatform(catalog *diagnosis.Catalog) {
 
 // RegisterPortConflict registers the port-conflict diagnose suite.
 func RegisterPortConflict(catalog *diagnosis.Catalog) {
-	// port-conflict suite available in darwin and linux only for now
-	if runtime.GOOS == "darwin" || runtime.GOOS == "linux" {
-		catalog.Register("port-conflict", func() []diagnosis.Diagnosis { return ports.DiagnosePortSuite() })
-	}
+	catalog.Register("port-conflict", func() []diagnosis.Diagnosis { return ports.DiagnosePortSuite() })
```

Comment:
I believe you can just use the following
```suggestion
	catalog.Register("port-conflict", ports.DiagnosePortSuite)
```